### PR TITLE
Todo Section - Dialogs

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/21352

*Description of changes:*
- Confirm DIalog for deleting draft (draft filing and draft application)
- Confirming Dialog for cancelling payment
- DeleteError Dialog
- CancelPaymentError Dialog
- Error Dialog for loading affiliation invitation
- Error Dialog for authorizing affiliation invitation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-28-720ty29n.web.app

**Review by hfekete — COMMENTED** [review]
In general OK.
Big question is is there not going to be internationalization (i18n) for these dialogs ?
Also 2 small ones

File: src/components/bcros/todo/List.vue [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+
 const prop = defineProps({
   todos: { type: Array<TodoItemI>, required: true }
 })
 
+const authorizeAffiliationError = ref(todosStore.authorizeAffiliationsErrors.length > 0)
+const loadAffiliationInvitationError = ref(todosStore.loadAffiliationsError.length > 0)
+
+const authorizeAffiliationErrorOptions: DialogOptionsI = {
+  title: 'Error updating affiliation invitation.',
```
[the remark this answers is not in the document]
Comment (kialj876) [reply src/components/bcros/todo/List.vue:12]: yes I agree I think this should all be from the lang file

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-28-720ty29n.web.app

File: src/components/bcros/todo/List.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (patrickpeinanw) [reply src/components/bcros/todo/List.vue]: Many texts in the dashboard are from the imported enums or functions, so the content can't be fully internationalized until those texts from the external source are updated. But I agree that we need to use the lang file in the new dashboard. 

I have updated the i18n stuff for the dialogs. There are many other places in the todo section that i18n need to be applied. I will change that later in a separate PR.

File: src/components/bcros/todo/List.vue [hunk 2]
```diff
@@ -28,9 +49,26 @@ const expand = (index: number, expanded: boolean) => {
...
+    id="todoList"
     class="flex flex-col"
     data-cy="todoItemList"
   >
+    <!-- error dialog (fetching affiliation request) -->
+    <BcrosDialog
+      attach="#todoList"
+      :display="loadAffiliationInvitationError"
+      :options="loadAffiliationInvitationErrorOptions"
+      @close="loadAffiliationInvitationError = false"
```
[the remark this answers is not in the document]
Comment (patrickpeinanw) [reply src/components/bcros/todo/List.vue]: Good point. Updated

File: src/components/bcros/todo/List.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (hfekete) [reply src/components/bcros/todo/List.vue]: Create a ticket if you dont have it, for the next PR

**Review by hfekete — APPROVED** [review]

**Comment by hfekete** [comment]
Looks good to me now! Thanks for updating texts.

**Pull request merged**